### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by TheAnonyUnknown**
> Added emojis for titles
> Warning section improved
> Layout adjusted as per mobile screen size, but pc layout got broke.

Do such that both screen sizes layout would work, if possible!
I can do, but need css stuff.

Help appreciated!

@XelXen @Bnyro

### CODE REVIEW

**Comment by Bnyro** [comment]
we should most likely use sth like 10vw for image widths to work on mobile and on desktop

**Comment by Bnyro** [comment]
but I didn't test yet whether it works on markdown (e.g. style="width: 30vw;")
feel free to do so :)

**Comment by TheAnonyUnknown** [comment]
> but I didn't test yet whether it works on markdown (e.g. style="width: 30vw;") feel free to do so :)

px works here 
Not sure abt vw

**Comment by Bnyro** [comment]
it's worth a try :)
at the current state of your pr the images feel way too small

**Comment by TheAnonyUnknown** [comment]
> it's worth a try :) at the current state of your pr the images feel way too small

Hmm, I'll check

**Comment by TheAnonyUnknown** [comment]
@Bnyro Have a look at new changes.
Seems fine to me though.

File: README.md [hunk 1]
```diff
@@ -7,25 +7,19 @@
...
.../android/en-US/images/phoneScreenshots/shot_3.png" width=32% alt="Player">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_3.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_4.png" width=32% alt="Channel">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_4.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_5.png" width=32% alt="Settings">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_5.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_6.png" width=32% alt="Subscriptions">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_6.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_7.png" width=32% alt="Subscriptions List">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_7.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_8.png" width=32% alt="Library">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_8.png) [<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_9.png" width=32% alt="Playlist">](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_9.png)
```
Comment (Bnyro) [inline README.md:18]: Why wrapping everything into one line?

File: README.md [hunk 2]
```diff
@@ -7,25 +7,19 @@
...
  
-[<img src="https://libre-tube.github.io/assets/fdrload.png" alt="Get it on F-Droid" height="80">](https://f-droid.org/en/packages/com.github.libretube/)
-[<img src="https://libre-tube.github.io/assets/izzyload.png" alt="Get it on IzzyOnDroid" height="80">](https://apt.izzysoft.de/fdroid/index/apk/com.github.libretube)
-[<img src="https://libre-tube.github.io/assets/ghload.png" alt="Get it on GitHub" height="80">](https://github.com/libre-tube/LibreTube/releases/latest)
-[<img src="https://libre-tube.github.io/assets/tgload.png" alt="Get it on GitHub" height="80">](https://t.me/LibreTube)
+[<img src="https://libre-tube.github.io/assets/fdrload.png" alt="Get it on F-Droid" width="45%">](https://f-droid.org/en/packages/com.github.libretube/) [<img src="https://libre-tube.github.io/assets/izzyload.png" alt="Get it on IzzyOnDroid" width="45%">](https://apt.izzysoft.de/fdroid/index/apk/com.github.libretube) [<img src="https://libre-tube.github.io/assets/ghload.png" alt="Get it on GitHub" width="45%">](https://github.com/libre-tube/LibreTube/releases/latest) [<img src="https://libre-tube.github.io/assets/tgload.png" alt="Get it on GitHub" width="45%">](https://t.me/LibreTube)
```
Comment (Bnyro) [inline README.md:10]: could u make them a bit smaller (like 30%)? they look way too big on desktop

Comment (TheAnonyUnknown) [reply README.md:10]: Cuz that way, both layouts' issue gets solved

@XelXen have a look sir

Comment (Bnyro) [reply README.md:10]: U dont trust me?

File: README.md (hunk 1 above)
Comment (TheAnonyUnknown) [reply README.md]: Fixed

**Review by Bnyro — APPROVED** [review]

**Comment by Bnyro** [comment]
image
they are still waaay too big on desktop

**Comment by TheAnonyUnknown** [comment]
> image they are still waaay too big on desktop

Ya, I tried using flex tag, layout prob solved.
But need to check sizing.

Will inform you when I'm done.

**Pull request merged**